packages/react/src/components/Combobox/Combobox.tsx

```diff
@@ -676,6 +686,24 @@ const Combobox = forwardRef<
             </span>
           )}
         </label>
+        <div>
+          {description && (
+            <span
+              className={classNames('Field__labelDescription', {
+                'Field__labelDescription--disabled': disabled
+              })}
```

Comment:
This should just be able to be `Field__description` based on https://github.com/dequelabs/cauldron/blob/d985ed72b148d58a6031ddb52dd5aed5ecbbc426/packages/react/src/components/TextField/index.tsx#L114-L118 and rendered as a `div` not a `span`.

```suggestion
              className="Field__description"
```